README.md

```diff
@@ -25,28 +35,27 @@ Azure service crates generated from [Azure REST API Specifications](https://gith
 
 🚨 WARNING 🚨: This project is under active development. Be aware that large breaking changes will happen before 1.0 is reached.
 
+### Historical releases
+
 This project is the successor to the `azure_sdk*` crates from [MindFlavor/AzureSDKForRust](https://github.com/MindFlavor/AzureSDKForRust). The crates have been renamed, so those older crates should be considered fully deprecated. See [history](HISTORY.md) for more details.
```

Comment:
Do you also want to include a link to the legacy branch holding the unofficial Azure SDK Releases?

Reply:
We haven't migrated yet and I don't think we should until we're ready to migrate `feature/track2` to `main`. People are still actively contributing to `main`. I think at that time this README should reflect that information, though, yes.